lib/uploader.ts

```diff
@@ -41,7 +39,12 @@ export class Uploader {
 
 	// Global upload queue
 	private _uploadQueue: Array<Upload> = []
-	private _jobQueue: PQueue = new PQueue({ concurrency: MAX_CONCURRENCY })
+	private _jobQueue: PQueue = new PQueue({
+		// Maximum number of concurrent uploads
+		// @ts-expect-error TS2339
+		concurrency: getCapabilities()?.files?.chunked_upload?.max_parallel_count ?? 5,
```

Comment:
I think `getCapabilities` always returns an object, so no need for the `?` on it :) 
```suggestion
		concurrency: getCapabilities().files?.chunked_upload?.max_parallel_count ?? 5,
```